kolibri/core/device/management/commands/provisiondevice.py

```diff
@@ -134,15 +138,17 @@ def create_superuser(username=None, password=None, interactive=False):
             confirm = get_user_response("Confirm password for the super user: ")
 
     if username and password:
-        if not FacilityUser.objects.filter(username__icontains=username).exists():
-            FacilityUser.objects.create_superuser(username, password)
+        try:
+            FacilityUser.objects.create_superuser(username, password, facility=facility)
             logger.info(
-                "Superuser created with username {username}.".format(username=username)
+                "Superuser created with username {username} in facility {facility}.".format(
+                    username=username, facility=facility
+                )
             )
-        else:
+        except ValidationError:
```

Comment:
Is it possible to run `provisiondevice` and run into this situation? I thought it would create a new empty Facility, so there wouldn't be any username collisions.

Reply:
yeah I think it could be, when the user somehow uses `provisiondevice` to create new superuser on existing facility instead of creating a new facility?

Reply:
Ohh, that's weird. Wasn't there a different command just for making superusers?

Reply:
There is a `createsuperuser` command that is a django builtin - which I think only creates a superuser on the default facility for the device.

Reply:
I think a better UX would be to reloop back to let the user pick a different name, but we can add that later.

Reply:
Oh, I keep forgetting that it's a django built-in command. Yeah, I can see a future version of `provisiondevice` being more of a generic CLI tool for doing stuff you would do on the Setup Wizard + Device pages.